vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBoxBase.java

```diff
@@ -355,20 +355,46 @@ protected void setFilter(String filter) {
     }
 
     /**
-     * Sets whether the component requires a value to be considered in a valid
-     * state.
+     * Sets whether the user is required to provide a value. When required, an
+     * indicator appears next to the label and the field invalidates if the
+     * value is cleared.
+     * <p>
+     * NOTE: The required indicator is only visible when the field has a label,
+     * see {@link #setLabel(String)}.
+     *
+     * @param required
+     *            true to make the field required, false otherwise
+     */
+    @Override
+    public void setRequiredIndicatorVisible(boolean required) {
+        super.setRequiredIndicatorVisible(required);
+    }
+
+    /**
+     * Gets whether the user is required to provide a value.
+     *
+     * @return true if the field is required, false otherwise
+     * @see #setRequiredIndicatorVisible(boolean)
+     */
+    @Override
+    public boolean isRequiredIndicatorVisible() {
+        return super.isRequiredIndicatorVisible();
+    }
+
+    /**
+     * Alias for {@link #isRequiredIndicatorVisible()}
      *
-     * @return {@code true} if the component requires a value to be valid
+     * @return true if the field is required, false otherwise
```

Comment:
We probably could keep using `{@code true}` for formatting in the IDE. Not a blocker though.

Reply:
Fixed